tests/integration/test_logcollector/test_statistics/test_statistics_macos.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+import sys
+from json import load
+import tempfile
+
+import pytest
+import wazuh_testing.tools.configuration as conf
+from wazuh_testing import logcollector
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools import LOGCOLLECTOR_STATISTICS_FILE
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.services import control_service
+
+from time import sleep

```

Comment:
```suggestion
import os
import pytest
import wazuh_testing.tools.configuration as conf

from json import load
from wazuh_testing import logcollector
from wazuh_testing.tools.configuration import load_wazuh_configurations
from wazuh_testing.tools import LOGCOLLECTOR_STATISTICS_FILE
```

Please, be careful with unused imports

Reply:
Changes applied.

`from wazuh_testing.tools.services import control_service`  is used in the test and cannot be deleted.